cmd/arc_summary/arc_summary2

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python2
```

Comment:
Did you consider substituting the shebang using autotools or whatever? The `/usr/bin/env` form is frowned upon in at least some Linux contexts, for example, Debian: https://www.debian.org/doc/packaging-manuals/python-policy/ch-python.html#s-interpreter_loc

Is `/usr/bin/env python2` actually desired on FreeBSD or is this just a work-around for the fact that it's (presumably) something other than `/usr/bin/python2`? That is, would you prefer this to be the direct path anyway?

Reply:
FreeBSD standard and often needed method is env https://www.freebsd.org/doc/en_US.ISO8859-1/books/porters-handbook/uses-shebangfix.html

So we may have to go with the autotools based fix.

Reply:
`/usr/bin/env python3` is the [suggested](https://docs.python.org/3/using/unix.html?highlight=shebang#miscellaneous) shebang for Python in general (likewise for python2), and conventional across platforms. On FreeBSD, Python is typically found under /usr/local/bin. I did hope to avoid autotools complexity, but I'm not opposed to investigating that option further. I can see arguments for either way.

Reply:
Personally I would prefer to have the env shebang in the tree for ease of development and let the various packaging scripts replace it as appropriate for their particular needs.

Reply:
That is a reasonable approach.